.github/workflows/build-modules.yml

```diff
@@ -94,7 +94,7 @@ jobs:
       - name: Upload Event
         uses: actions/upload-artifact@v4
         with:
-          name: Event File
+          name: Event File ${{ matrix.module }}
```

Comment:
With this change we will also have to update the action that handles these files:

https://github.com/SEKOIA-IO/automation-library/blob/4bcbe6f474b64c7885f41d41718fcfc58c1ad178/.github/workflows/test_results.yml#L26-L32